tonto/routers_tonto.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import cowboys
+from tonto import crud, schemas
+from settings import bd_coneccion
+
+router = APIRouter()
+
+
+@router.get ("/tonto", response_model = cowboys.schemas.CrearCowboyRespuesta)
+def seleccionar_tonto (db: Session = Depends (bd_coneccion.get_db)):
+    cowboy  = crud.tonto_random (db)
+
+    if cowboy is None:
+        raise HTTPException (status_code = 404, detail = "No existe ese cowboy")
+
+    return cowboy
```

Comment:
Según la API actual que tenemos que copiar para que la pagina siga fucionando igual, este endpoint debería devolver la lista de tontos cada uno con sus titulos.

![image](https://github.com/The-Cowboys/tonto-apy/assets/25726643/209c8617-9eb9-450b-ba26-d7cd5dc3c93d)